sympy/polys/numberfields/subfield.py

```diff
@@ -340,32 +340,52 @@ def primitive_element(extension, x=None, *, ex=False, polys=False):
     """
     if not extension:
         raise ValueError("Cannot compute primitive element for empty extension")
+    extension = [_sympify(ext) for ext in extension]
 
     if x is not None:
         x, cls = sympify(x), Poly
     else:
         x, cls = Dummy('x'), PurePoly
 
+    coeffs = []
+    reps = []
```

Comment:
reps is needed only in the ex=True case.

Reply:
Yeah, reps is being returned only during `ex=True` case i think.